lib/OpenQA/Constants.pm

```diff
@@ -99,7 +99,7 @@ use constant VIDEO_FILE_NAME_REGEX => qr/^.*\/video\.[^\/]*$/;
 use constant FRAGMENT_REGEX => qr'(#([-?/:@.~!$&\'()*+,;=\w]|%[0-9a-fA-F]{2})*)*';
 
 use constant JOBS_OVERVIEW_SEARCH_CRITERIA =>
-  (qw(distri version flavor build test modules modules_result module_re group groupid id));
+  (qw(arch distri version flavor build test modules modules_result module_re group groupid id machine));
```

Comment:
We commonly order it like this so please write like this

```suggestion
  (qw(distri version flavor arch build test machine modules modules_result module_re group groupid id));
```

Reply:
Thanks for the review, makes sense to me!